# Libgda Spanish Translation
# Copyright (C) YEAR Free Software Foundation, Inc.
# Rodrigo Moya <rmoya@chez.com>, 1999.
# Ismael Olea <olea@hispafuentes.com>, 2000
# Carlos Perelló Marín <carlos@gnome-db.org>, 2001
# Antonio de la Torre <adltorre@terra.es>, 2001
#
msgid ""
msgstr ""
"Project-Id-Version: gnome-db \n"
"POT-Creation-Date: 2001-09-16 22:13+0200\n"
"PO-Revision-Date: 2001-03-14 01:06+0100\n"
"Last-Translator: Antonio de la Torre <adltorre@terra.es>\n"
"Language-Team: Spanish <es@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=iso-8859-1\n"
"Content-Transfer-Encoding: 8bit\n"

#: lib/gda-client/gda-connection-pool.c:268
#, c-format
msgid "could not open connection to %s"
msgstr "No se pudo abrir la conexión con %s"

#: lib/gda-client/gda-connection-pool.c:281
#, c-format
msgid "Data source %s not found"
msgstr "No se encontró la fuentes de datos %s"

#: lib/gda-client/gda-connection.c:447
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "La base de datos '%s' no se ha encontrado en el sistema"

#: lib/gda-client/gda-connection.c:451 lib/gda-client/gda-connection.c:468
msgid "[GDA Client Library]"
msgstr ""

#: lib/gda-client/gda-connection.c:467
msgid "Could not open CORBA factory"
msgstr "No se ha podido abrir la \"factory\" de CORBA"

#: lib/gda-client/gda-field.c:228
msgid "<NULL>"
msgstr "<VACÍO>"

#: lib/gda-client/gda-field.c:233 lib/gda-client/gda-field.c:236
msgid "<Unknown GDA Type(NULL)>"
msgstr "<Tipo GDA desconocido (VACÍO)>"

#: lib/gda-client/gda-field.c:249
msgid "TRUE"
msgstr "SI"

#: lib/gda-client/gda-field.c:251
msgid "FALSE"
msgstr "NO"

#: lib/gda-common/gda-corba.c:88
#, c-format
msgid "CORBA System exception: %s"
msgstr "Excepción CORBA del sistema: %s"

#: lib/gda-common/gda-corba.c:123
msgid "True"
msgstr "Verdad"

#: lib/gda-common/gda-corba.c:124
msgid "False"
msgstr "Falso"

#: lib/gda-common/gda-error.c:128
#, c-format
msgid "%s: The server didn't respond."
msgstr "%s: El servidor no contesta."

#: lib/gda-common/gda-error.c:135
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Ha ocurrido un error en el sistema CORBA."

#: lib/gda-common/gda-error.c:152
#, fuzzy
msgid "[CORBA User Exception]"
msgstr "Excepción CORBA de usuario: %s"

#: lib/gda-common/gda-error.c:201
msgid "Unknown CORBA exception for connection"
msgstr ""

#: lib/gda-common/gda-init.c:55
msgid "Attempt to initialize an already initialized client"
msgstr "Se ha intentado inicializar un cliente que ya lo estaba"

#: lib/gda-common/gda-init.c:64 lib/gda-server/gda-server-init.c:121
#, fuzzy
msgid "Could not initialize Bonobo"
msgstr "No pudo iniciarse ct_diag()"

#: lib/gda-common/gda-log.c:48
msgid "Saving log...x"
msgstr "Guardando el log...x"

#. compose message
#: lib/gda-common/gda-log.c:68
msgid "ERROR: "
msgstr "ERROR: "

#: lib/gda-common/gda-log.c:68
msgid "MESSAGE: "
msgstr "MENSAJE: "

#: lib/gda-common/gda-thread.c:168
msgid "thread is already running"
msgstr "el hilo ya se está ejecutando"

#: lib/gda-common/gda-util.c:113
#, fuzzy, c-format
msgid "Could not open file %s"
msgstr "No pudo ubicar local para '%s'"

#: lib/gda-common/gda-util.c:138
#, fuzzy, c-format
msgid "Could not create file %s"
msgstr "No pudo asignarse campo"

#: lib/gda-common/gda-xml-database.c:166
msgid "bad formed document"
msgstr ""

#: lib/gda-common/gda-xml-database.c:267
#, c-format
msgid "table %s already exists"
msgstr "la tabla %s ya existe"

#: lib/gda-common/gda-xml-database.c:449
#, c-format
msgid "%p is not a valid table"
msgstr "%p no es una tabla válida"

#: lib/gda-common/gda-xml-item.c:385
#, fuzzy, c-format
msgid "Item with id %s not found"
msgstr "No se encontró la fuentes de datos %s"

#: lib/gda-server/gda-server-connection.c:404
#, fuzzy
msgid "Connection being closed"
msgstr "La conexión parece que ha terminado"

#: lib/gda-server/gda-server-connection.c:407
msgid "This connection is being closed, so all listeners are released"
msgstr ""

#: lib/gda-server/gda-server-error.c:41
msgid "Could not get pointer to server implementation"
msgstr "No se puede obtener un puntero a la implementación del servidor"

#: lib/gda-server/gda-server-init.c:43 lib/gda-server/gda-server-init.c:49
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Recibida la señal %d, cerrando"

#: lib/gda-server/gda-server-init.c:53
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Recibida la señal %d. Ignorándola"

#: lib/gda-server/gda-server-init.c:104
msgid "Attempt to initialize an already initialized provider"
msgstr "Se ha intentado inicializar un proveedor que ya se había inicializado"

#: lib/gda-server/gda-server-recordset.c:52
#: lib/gda-server/gda-server-recordset.c:60
#: lib/gda-server/gda-server-recordset.c:100
#: lib/gda-server/gda-server-recordset.c:108
#, c-format
msgid "%s not implemented"
msgstr "%s no está implementado"

#: lib/gda-server/gda-server.c:128
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Iniciando el proveedor %s sin funciones de implementación"

#: lib/gda-server/gda-server.c:214
#, c-format
msgid "CORBA system exception %s"
msgstr "Excepción CORBA del sistema %s"

#: lib/gda-server/gda-server.c:218
#, c-format
msgid "CORBA user exception: %s"
msgstr "Excepción CORBA de usuario: %s"

#: providers/gda-default-server/GNOME_GDA_Provider_Default.oaf.in.h:1
#, fuzzy
msgid "Default provider connection factory"
msgstr "No se pudo abrir la conexión con %s"

#: providers/gda-default-server/GNOME_GDA_Provider_Default.oaf.in.h:2
#, fuzzy
msgid "GDA Default Datasource Access"
msgstr "Acceso de Fuente de datos GDA para ODBC"

#: providers/gda-default-server/gda-default-connection.c:127
msgid "You must specify a directory for the database to be stored"
msgstr ""

#: providers/gda-default-server/gda-default-connection.c:235
#: providers/gda-mysql-server/gda-mysql-connection.c:282
#: providers/gda-oracle-server/gda-oracle-connection.c:334
#: providers/gda-postgres-server/gda-postgres-connection.c:486
#, fuzzy
msgid "Unknown schema type"
msgstr "Tipo Oracle %ld desconocido"

#: providers/gda-default-server/main-default.c:69
#, fuzzy
msgid "Could not register Default provider implementation"
msgstr "No se ha podido registrar la implementación del proveedor de Correo"

#: providers/gda-ldap-server/GNOME_GDA_Provider_LDAP.oaf.in.h:1
msgid "GDA Datasource Access for LDAP"
msgstr "Fuente de datos GDA para LDAP"

#: providers/gda-ldap-server/GNOME_GDA_Provider_LDAP.oaf.in.h:2
msgid "LDAP provider connection factory"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:288
#: providers/gda-ldap-server/gda-ldap-connection.c:293
#: providers/gda-mdb-server/gda-mdb-connection.c:372
#: providers/gda-mysql-server/gda-mysql-connection.c:486
#: providers/gda-odbc-server/gda-odbc-connection.c:624
#: providers/gda-odbc-server/gda-odbc-connection.c:657
#: providers/gda-oracle-server/gda-oracle-connection.c:479
#: providers/gda-postgres-server/gda-postgres-connection.c:767
#: providers/gda-sybase-server/gda-sybase-error.c:132
#: providers/gda-tds-server/gda-tds-error.c:123
#, c-format
msgid "error '%s' at %s"
msgstr "error '%s' en %s"

#. gda_error_set_help_file (error, _("Not available"));
#: providers/gda-interbase-server/gda-interbase-connection.c:294
#: providers/gda-ldap-server/gda-ldap-connection.c:298
#: providers/gda-ldap-server/gda-ldap-connection.c:299
#: providers/gda-mdb-server/gda-mdb-connection.c:377
#: providers/gda-mysql-server/gda-mysql-connection.c:491
#: providers/gda-mysql-server/gda-mysql-connection.c:492
#: providers/gda-odbc-server/gda-odbc-connection.c:626
#: providers/gda-odbc-server/gda-odbc-connection.c:628
#: providers/gda-odbc-server/gda-odbc-connection.c:659
#: providers/gda-odbc-server/gda-odbc-connection.c:661
#: providers/gda-oracle-server/gda-oracle-connection.c:484
#: providers/gda-oracle-server/gda-oracle-connection.c:485
#: providers/gda-postgres-server/gda-postgres-connection.c:775
#: providers/gda-postgres-server/gda-postgres-connection.c:776
#: providers/gda-primebase-server/gda-primebase-connection.c:375
#: providers/gda-primebase-server/gda-primebase-connection.c:376
#: providers/gda-sybase-server/gda-sybase-error.c:142
#: providers/gda-tds-server/gda-tds-error.c:132
#: providers/gda-tds-server/gda-tds-error.c:133
msgid "Not available"
msgstr "No está disponible"

#: providers/gda-interbase-server/gda-interbase-connection.c:295
#: providers/gda-ldap-server/gda-ldap-connection.c:300
#: providers/gda-mdb-server/gda-mdb-connection.c:378
#: providers/gda-mysql-server/gda-mysql-connection.c:493
#: providers/gda-oracle-server/gda-oracle-connection.c:486
#: providers/gda-postgres-server/gda-postgres-connection.c:777
#: providers/gda-primebase-server/gda-primebase-connection.c:377
#: providers/gda-sybase-server/gda-sybase-error.c:143
#: providers/gda-tds-server/gda-tds-error.c:134
msgid "error"
msgstr "error"

#: providers/gda-ldap-server/main-ldap.c:73
msgid "Could not register LDAP provider implementation"
msgstr "No se ha podido registrar la implementación del proveedor LDAP"

#: providers/gda-interbase-server/GNOME_GDA_Provider_InterBase.oaf.in.h:1
msgid "GDA Datasource Access for Interbase"
msgstr "Fuente de datos GDA para Interbase"

#: providers/gda-interbase-server/GNOME_GDA_Provider_InterBase.oaf.in.h:2
msgid "InterBase provider connection factory"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:286
#, c-format
msgid "Error code %ld"
msgstr "Código de error %ld"

#: providers/gda-interbase-server/main-interbase.c:74
msgid "Could not register INTERBASE provider implementation"
msgstr "No se ha podido registrar la implementación del proveedor INTERBASE"

#: providers/gda-mail-server/GNOME_GDA_Provider_Mail.oaf.in.h:1
msgid "GDA Datasource Access for Mail"
msgstr "Fuente de datos GDA para Correo"

#: providers/gda-mail-server/GNOME_GDA_Provider_Mail.oaf.in.h:2
msgid "Mail provider connection factory"
msgstr ""

#: providers/gda-mail-server/gda-mail-connection.c:107
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-sybase-server/gda-sybase-connection.c:290
#: providers/gda-tds-server/gda-tds-connection.c:278
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "SCHEMA_QTYPE %d no manejado"

#: providers/gda-mail-server/main-mail.c:74
msgid "Could not register Mail provider implementation"
msgstr "No se ha podido registrar la implementación del proveedor de Correo"

#: providers/gda-mdb-server/GNOME_GDA_Provider_MDB.oaf.in.h:1
msgid "GDA Datasource Access for MDB (MS Access) files"
msgstr "Fuente de datos GDA para archivos MDB (MS Access)"

#: providers/gda-mdb-server/GNOME_GDA_Provider_MDB.oaf.in.h:2
msgid "MDB provider connection factory"
msgstr ""

#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:144
#: providers/gda-sybase-server/gda-sybase-command.c:141
#: providers/gda-tds-server/gda-tds-command.c:120
#, c-format
msgid "executing command '%s'"
msgstr "ejecutando el comando '%s'"

#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:149
#, c-format
msgid "info '%s'"
msgstr "info '%s'"

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr ""
"%s:%d: No puede facilitar un UNIX_SOCKET si también da un HOST o un PORT. "
"Elimine el UNIX_SOCKET, o elimine las opciones HOST y el PORT"

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr ""
"%s:%d: Debe de facilitar el nombre del archivo de la base de datos de MS "
"Access"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr "Abriendo conexión con  archivo=%s, usuario=%s, contraseña=%s"

#: providers/gda-mdb-server/gda-mdb-connection.c:427
msgid "mysql_recset is NULL"
msgstr "mysql_recset está VACÍO"

#: providers/gda-mdb-server/gda-mdb-connection.c:469
#: providers/gda-mysql-server/gda-mysql-connection.c:838
#: providers/gda-sybase-server/gda-sybase-connection.c:469
#: providers/gda-tds-server/gda-tds-connection.c:448
msgid "No table name given for SCHEMA_COLS"
msgstr "No ha se ha dado el nombre de una tabla para SCHEMA_COLS"

#: providers/gda-mdb-server/main-mdb.c:72
#: providers/gda-mysql-server/main-mysql.c:74
msgid "Could not register MySQL provider implementation"
msgstr "No se ha podido registrar la implementación del proveedor MySQL"

#: providers/gda-mysql-server/GNOME_GDA_Provider_MySQL.oaf.in.h:1
msgid "GDA Datasource Access for MySQL"
msgstr "Fuente de datos GDA para MySQL"

#: providers/gda-mysql-server/GNOME_GDA_Provider_MySQL.oaf.in.h:2
msgid "MySQL provider connection factory"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:174
#, fuzzy
msgid ""
"You cannot provide a UNIX_SOCKET if you also provide either a HOST or a "
"PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT options"
msgstr ""
"%s:%d: No puede facilitar un UNIX_SOCKET si también da un HOST o un PORT. "
"Elimine el UNIX_SOCKET, o elimine las opciones HOST y el PORT"

#: providers/gda-mysql-server/gda-mysql-connection.c:190
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%s, port=%s, unix_socket=%"
"s"
msgstr ""
"Abriendo conexión con el usuario=%s, contraseña=%s, sistema=%s, puerto=%s, "
"unix_socket=%s"

#: providers/gda-mysql-server/gda-mysql-connection.c:616
msgid "Unknown"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:649
msgid "No MYSQL_Connection for GdaServerConnection"
msgstr ""

#: providers/gda-odbc-server/GNOME_GDA_Provider_ODBC.oaf.in.h:1
msgid "GDA Datasource Access for ODBC"
msgstr "Acceso de Fuente de datos GDA para ODBC"

#: providers/gda-odbc-server/GNOME_GDA_Provider_ODBC.oaf.in.h:2
msgid "ODBC provider connection factory"
msgstr ""

#: providers/gda-odbc-server/gda-odbc-connection.c:277
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: SCHEMA_QTYPE %d no manejado\n"

#: providers/gda-odbc-server/gda-odbc-connection.c:641
#: providers/gda-odbc-server/gda-odbc-connection.c:642
#: providers/gda-odbc-server/gda-odbc-connection.c:643
#: providers/gda-odbc-server/gda-odbc-connection.c:674
#: providers/gda-odbc-server/gda-odbc-connection.c:675
#: providers/gda-odbc-server/gda-odbc-connection.c:676
msgid "no text"
msgstr "ningún texto"

#: providers/gda-odbc-server/main-odbc.c:73
msgid "Could not register ODBC provider implementation"
msgstr "No pudo registrase la implementación del proveedor ODBC"

#: providers/gda-oracle-server/GNOME_GDA_Provider_Oracle.oaf.in.h:1
msgid "GDA Datasource Access for Oracle"
msgstr "Fuente de datos GDA para Oracle"

#: providers/gda-oracle-server/GNOME_GDA_Provider_Oracle.oaf.in.h:2
msgid "Oracle provider connection factory"
msgstr ""

#: providers/gda-oracle-server/gda-oracle-command.c:191
msgid "Could not allocate statement handle"
msgstr "No pudo localizarse el manejador de orden"

#: providers/gda-oracle-server/gda-oracle-command.c:197
msgid "command object being created without associated connection"
msgstr "objeto comando creándose sin conexión asociada"

#: providers/gda-oracle-server/gda-oracle-command.c:238
#, c-format
msgid "Executing command '%s'"
msgstr "Ejecutando comando '%s'"

#: providers/gda-oracle-server/gda-oracle-connection.c:117
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "error en OCIHandleAlloc() HTYPE_SVCCTX"

#: providers/gda-oracle-server/gda-oracle-connection.c:123
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "error en OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:129
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "error en OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:418
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "Tipo Oracle %ld desconocido"

#: providers/gda-oracle-server/gda-oracle-recordset.c:118
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "Tipo de datos Oracle desconocido en fill_field_values"

#: providers/gda-oracle-server/gda-oracle-recordset.c:156
msgid "recordset object being created without associated connection"
msgstr "Objeto recordset creándose sin conexión asociada"

#: providers/gda-oracle-server/main-oracle.c:69
msgid "Could not register ORACLE provider implementation"
msgstr "No pudo registrarse la implementación del proveedor de Oracle"

#: providers/gda-postgres-server/GNOME_GDA_Provider_Postgres.oaf.in.h:1
msgid "GDA Datasource Access for Postgres"
msgstr "Acceso de la Fuente de Datos GDA para Postgres"

#: providers/gda-postgres-server/GNOME_GDA_Provider_Postgres.oaf.in.h:2
msgid "Postgres provider connection factory"
msgstr ""

#: providers/gda-postgres-server/gda-postgres-connection.c:888
#: providers/gda-postgres-server/gda-postgres-connection.c:1005
msgid "postgres_recset is NULL"
msgstr "postgres_recset está VACÍO"

#: providers/gda-postgres-server/main-postgres.c:71
msgid "Could not register PostgreSQL provider implementation"
msgstr "No pudo registrarse la implementación del proveedor PostgreSQL"

#: providers/gda-sybase-server/GNOME_GDA_Provider_Sybase.oaf.in.h:1
msgid "GDA Datasource Access for Sybase"
msgstr "Acceso de la Fuente de Datos GDA para Sybase"

#: providers/gda-sybase-server/GNOME_GDA_Provider_Sybase.oaf.in.h:2
msgid "Sybase provider connection factory"
msgstr ""

#.      gda_server_error_make(error, 0, cnc, "connection seems to be dead");
#: providers/gda-sybase-server/gda-sybase-command.c:136
#: providers/gda-tds-server/gda-tds-command.c:116
msgid "Connection seems to be dead"
msgstr "La conexión parece que ha terminado"

#: providers/gda-sybase-server/gda-sybase-command.c:153
#: providers/gda-tds-server/gda-tds-command.c:130
msgid "Filling command structure failed\n"
msgstr "Error al rellenar la estructura de comando\n"

#: providers/gda-sybase-server/gda-sybase-command.c:162
#: providers/gda-tds-server/gda-tds-command.c:137
msgid "sending cmd failed"
msgstr "error al enviar cmd"

#: providers/gda-sybase-server/gda-sybase-command.c:169
#: providers/gda-tds-server/gda-tds-command.c:144
msgid "Allocating recordset failed"
msgstr "Error al adjudicar recordset"

#: providers/gda-sybase-server/gda-sybase-command.c:181
#: providers/gda-tds-server/gda-tds-command.c:153
msgid "Getting recordset userdata failed"
msgstr "Error al obtener datos de usuario de recordset"

#: providers/gda-sybase-server/gda-sybase-command.c:241
#: providers/gda-tds-server/gda-tds-command.c:210
msgid "Command execution finished, all results processed"
msgstr "Ejecución de comando finalizada. todos los resultados procesados"

#: providers/gda-sybase-server/gda-sybase-command.c:245
#: providers/gda-tds-server/gda-tds-command.c:214
msgid "Command failed"
msgstr "Comando fallido"

#: providers/gda-sybase-server/gda-sybase-command.c:250
#: providers/gda-tds-server/gda-tds-command.c:219
msgid "Execution of resultless command succeeded"
msgstr "Ejecución correcta de comando resultless"

#: providers/gda-sybase-server/gda-sybase-command.c:268
#: providers/gda-tds-server/gda-tds-command.c:237
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_results() devolvió %x"

#: providers/gda-sybase-server/gda-sybase-command.c:280
#: providers/gda-tds-server/gda-tds-command.c:249
msgid "No further results, command completly succeeded."
msgstr "No más resultados, comando correcto"

#: providers/gda-sybase-server/gda-sybase-command.c:285
#: providers/gda-tds-server/gda-tds-command.c:254
msgid "An error occured during command execution."
msgstr "Un error ha ocurrido durante la ejecución del comando."

#: providers/gda-sybase-server/gda-sybase-command.c:291
#: providers/gda-tds-server/gda-tds-command.c:258
msgid "Cmd execution returned unknown result, may be an error"
msgstr ""
"La ejecución de Cmd devolvió un resultado desconocido, puede ser un error"

#: providers/gda-sybase-server/gda-sybase-command.c:304
#: providers/gda-tds-server/gda-tds-command.c:269
msgid "No data returned, freeing recordset structure"
msgstr "No hay datos devueltos, liberando estructura recordset"

#: providers/gda-sybase-server/gda-sybase-command.c:336
#: providers/gda-sybase-server/gda-sybase-command.c:348
#: providers/gda-tds-server/gda-tds-command.c:300
#: providers/gda-tds-server/gda-tds-command.c:310
msgid "Freeing command structure failed"
msgstr "Error al liberar la estructura de comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:176
#, fuzzy
msgid "Not enough memory for sybase_Connection"
msgstr "No hay suficiente memoria para primebase_Error structure."

#: providers/gda-sybase-server/gda-sybase-connection.c:458
#: providers/gda-tds-server/gda-tds-connection.c:437
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: table_name '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:475
#: providers/gda-tds-server/gda-tds-connection.c:454
msgid "Could not allocate command structure"
msgstr "No pudo asignarse estructura de comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:498
#: providers/gda-tds-server/gda-tds-connection.c:477
msgid "Could not allocate enough memory for query"
msgstr "No pudo asignarse suficiente memoria para la búsqueda"

#.          gda_server_error_make(error, NULL, cnc, __PRETTY_FUNCTION__);
#: providers/gda-sybase-server/gda-sybase-connection.c:513
#: providers/gda-tds-server/gda-tds-connection.c:492
msgid "Could not get recordset in schema_cols"
msgstr "No pudo obtenerse recordset en schema_cols"

#: providers/gda-sybase-server/gda-sybase-connection.c:562
#: providers/gda-tds-server/gda-tds-connection.c:541
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr "schema_procs: restricción no válida tipo %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:572
#: providers/gda-tds-server/gda-tds-connection.c:551
msgid "Could not allocate cmd structure"
msgstr "No pudo asignarse estructura cmd"

#: providers/gda-sybase-server/gda-sybase-connection.c:638
#: providers/gda-tds-server/gda-tds-connection.c:617
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr "schema_tables: restricción no válida tipo %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:700
#: providers/gda-tds-server/gda-tds-connection.c:679
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr "Tipo de restricción no válida o no soportada: %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:743
#: providers/gda-tds-server/gda-tds-connection.c:722
msgid "Could not allocate GdaServerCommand"
msgstr "No pudo asignarse GdaServerCommand"

#: providers/gda-sybase-server/gda-sybase-connection.c:774
#: providers/gda-sybase-server/gda-sybase-connection.c:779
#: providers/gda-tds-server/gda-tds-connection.c:752
#: providers/gda-tds-server/gda-tds-connection.c:757
#, c-format
msgid "connection status: %x"
msgstr "Estado de conexión: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:775
#: providers/gda-tds-server/gda-tds-connection.c:753
#, c-format
msgid "return code: %x"
msgstr "código devuelto: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:813
#: providers/gda-tds-server/gda-tds-connection.c:788
msgid "attempting reconnection"
msgstr "intentando reconexión"

#: providers/gda-sybase-server/gda-sybase-connection.c:943
#, c-format
msgid "Reconnected to '%s'"
msgstr "Reconectado a '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:945
#: providers/gda-tds-server/gda-tds-connection.c:890
#, c-format
msgid "Connected to %s"
msgstr "Conectado a %s"

#: providers/gda-sybase-server/gda-sybase-connection.c:957
#: providers/gda-tds-server/gda-tds-connection.c:904
#, c-format
msgid "Selected database '%s'"
msgstr "Base de datos '%s'seleccionada"

#: providers/gda-sybase-server/gda-sybase-connection.c:986
#: providers/gda-tds-server/gda-tds-connection.c:930
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr "%s: No pudo asignarse estructura cmd"

#: providers/gda-sybase-server/gda-sybase-connection.c:995
#: providers/gda-tds-server/gda-tds-connection.c:939
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: No pudo asignarse memoria para la serie cmd"

#: providers/gda-sybase-server/gda-sybase-connection.c:1004
#: providers/gda-tds-server/gda-tds-connection.c:947
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: ct_command fallido"

#: providers/gda-sybase-server/gda-sybase-connection.c:1015
#: providers/gda-tds-server/gda-tds-connection.c:957
#, c-format
msgid "%s: sending command failed"
msgstr "%s: error al enviar comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:1026
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: error al seleccionar base de datos"

#: providers/gda-sybase-server/gda-sybase-connection.c:1043
#, c-format
msgid "%s: use command failed"
msgstr "%s: error de comando uso"

#: providers/gda-sybase-server/gda-sybase-connection.c:1071
#: providers/gda-tds-server/gda-tds-connection.c:1020
msgid "Could not allocate locale structure"
msgstr "No pudo asignarse estructura local"

#: providers/gda-sybase-server/gda-sybase-connection.c:1077
#: providers/gda-tds-server/gda-tds-connection.c:1026
#, c-format
msgid "Could not set locale to '%s'"
msgstr "No pudo ubicar local para '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1084
#: providers/gda-tds-server/gda-tds-connection.c:1033
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr "No pudo configurar local para ser '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1090
#: providers/gda-tds-server/gda-tds-connection.c:1039
msgid "Could not drop locale structure"
msgstr "No pudo darse de baja la estructura local"

#: providers/gda-sybase-server/gda-sybase-connection.c:1104
#: providers/gda-tds-server/gda-tds-connection.c:1052
#, c-format
msgid "Fatal error: %s\n"
msgstr "Error fatal: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:78
#, c-format
msgid "gda_sybase_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-sybase-server/gda-sybase-error.c:90
#: providers/gda-tds-server/gda-tds-error.c:81
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr ""
"Ninguna señal de error está fijada, sin embargo %s está llamado. Esto puede "
"ser una fallo"

#: providers/gda-sybase-server/gda-sybase-error.c:166
msgid "cs_diag() initialized"
msgstr "cs_diag() iniciado"

#: providers/gda-sybase-server/gda-sybase-error.c:170
msgid "Could not initialize cs_diag()"
msgstr "No pudo iniciarse cs_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:180
msgid "ct_diag() initialized"
msgstr "ct_diag() iniciado"

#: providers/gda-sybase-server/gda-sybase-error.c:184
msgid "Could not initialize ct_diag()"
msgstr "No pudo iniciarse ct_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:220
#, c-format
msgid "%s could not find server implementation"
msgstr "%s no pudo encontrar la implementación del servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:227
#, c-format
msgid "%s could not get cnc userdata"
msgstr "%s no pudo obtener datos de usuario cnc"

#: providers/gda-sybase-server/gda-sybase-error.c:239
msgid "Could not get # of messages"
msgstr "No pudo obtener # de mensajes"

#: providers/gda-sybase-server/gda-sybase-error.c:246
#, c-format
msgid "%d ctlib messages to fetch"
msgstr "%d mensajes ctlib para retomar"

#: providers/gda-sybase-server/gda-sybase-error.c:251
#, c-format
msgid "%d ctlib message to fetch"
msgstr "%d mensaje ctlib para retomar"

#: providers/gda-sybase-server/gda-sybase-error.c:289
msgid "error is nullpointer in gda_sybase_messages_handle"
msgstr "el error no tiene indicador en gda_sybase_messages_handle"

#: providers/gda-sybase-server/gda-sybase-error.c:315
#: providers/gda-sybase-server/gda-sybase-error.c:387
#: providers/gda-sybase-server/gda-sybase-error.c:434
msgid "Could not get # of server messages"
msgstr "No pudo obtenerse # mensajes del servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:323
#, c-format
msgid "Fetching %d client messages"
msgstr "Retomando %d mensajes del cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:326
#, c-format
msgid "Fetching %d client message"
msgstr "Retomando %d mensaje del cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:335
#, c-format
msgid "Could not get client msg # %d"
msgstr "No pudo obtenerse mensaje # %d del cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:354
msgid "Could not get # of client messages"
msgstr "No pudo obtenerse # mensajes del cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:362
msgid "Could not clear client messages"
msgstr "No pudo vaciarse los mensajes del cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:395
#, c-format
msgid "Fetching %d server messages"
msgstr "Retomando %d mensajes del servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:398
#, c-format
msgid "Fetching %d server message"
msgstr "Retomando %d mensajes del servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:407
#, c-format
msgid "Could not get server msg # %d"
msgstr "No pudo obtenerse mensaje # %d del servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:442
msgid "Could not clear server messages"
msgstr "No pudo vaciarse los mensajes del servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:464
#: providers/gda-sybase-server/gda-sybase-error.c:504
msgid "Could not get # of cslib messages"
msgstr "No pudo obtenerse # mensajes cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:472
#, c-format
msgid "Fetching %d cslib messages"
msgstr "Retomando %d mensajes cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:475
#, c-format
msgid "Fetching %d cslib message"
msgstr "Retomando %d mensaje cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:485
#, c-format
msgid "Could not get cslib msg # %d"
msgstr "No pudo obtenerse mensaje # %d cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:512
msgid "Could not clear cslib messages"
msgstr "No pudo vaciarse los mensajes cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:646
#: providers/gda-tds-server/gda-tds-error.c:269
msgid "Server message:"
msgstr "Mensaje del servidor:"

#: providers/gda-sybase-server/gda-sybase-error.c:654
#: providers/gda-tds-server/gda-tds-error.c:277
msgid "Client message:"
msgstr "Mensaje del cliente:"

#: providers/gda-sybase-server/gda-sybase-error.c:661
#: providers/gda-tds-server/gda-tds-error.c:284
msgid "CS-Library error:"
msgstr "Error CS-Library:"

#: providers/gda-sybase-server/gda-sybase-recordset.c:203
#: providers/gda-tds-server/gda-tds-recordset.c:194
#, c-format
msgid "fetch error at row %d"
msgstr "Retomar error en la fila %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:210
#: providers/gda-tds-server/gda-tds-recordset.c:201
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA en la fila %d; cancelando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:218
#: providers/gda-tds-server/gda-tds-recordset.c:208
#, c-format
msgid "%s: Error (%d)"
msgstr "%s: Error (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:391
#: providers/gda-tds-server/gda-tds-recordset.c:382
msgid "Failed fetching # of columns"
msgstr "Error retomando # de columnas"

#: providers/gda-sybase-server/gda-sybase-recordset.c:398
#: providers/gda-tds-server/gda-tds-recordset.c:390
#, c-format
msgid "Counted %d columns"
msgstr "contadas %d columnas"

#: providers/gda-sybase-server/gda-sybase-recordset.c:405
#: providers/gda-tds-server/gda-tds-recordset.c:397
#, c-format
msgid "%s could not allocate datafmt info"
msgstr "%s no pudo asignar información datafmt"

#: providers/gda-sybase-server/gda-sybase-recordset.c:415
#: providers/gda-tds-server/gda-tds-recordset.c:407
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s no pudo asignar campos de datos"

#: providers/gda-sybase-server/gda-sybase-recordset.c:433
#: providers/gda-tds-server/gda-tds-recordset.c:425
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe en la columna # %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:440
#: providers/gda-tds-server/gda-tds-recordset.c:432
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "Error ct_describe en la columna # %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:447
#: providers/gda-tds-server/gda-tds-recordset.c:439
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "ct_describe(%d): '%s', %d (%s)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:459
#: providers/gda-tds-server/gda-tds-recordset.c:450
msgid "could not allocate data holder"
msgstr "no pudo asignarse sustento de datos"

#: providers/gda-sybase-server/gda-sybase-recordset.c:472
#: providers/gda-tds-server/gda-tds-recordset.c:463
msgid "could not ct_bind data holder to column"
msgstr "no pudo asignarse sustento de datos ct_bind para la columna"

#: providers/gda-sybase-server/gda-sybase-recordset.c:479
#: providers/gda-tds-server/gda-tds-recordset.c:470
msgid "illegal type detected, aborting"
msgstr "tipo ilegal detectado, abortando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:486
#: providers/gda-tds-server/gda-tds-recordset.c:477
msgid "could not allocate field"
msgstr "No pudo asignarse campo"

#: providers/gda-sybase-server/main-sybase.c:78
msgid "Could not register gda-sybase provider implementation"
msgstr "No pudo registrarse implementación de proveedor gda-sybase"

#: providers/gda-primebase-server/GNOME_GDA_Provider_Primebase.oaf.in.h:1
msgid "GDA Datasource Access for primebase"
msgstr "Acceso de Fuente de datos GDA para primebase"

#: providers/gda-primebase-server/gda-primebase-connection.c:157
#, c-format
msgid "Could not connect to '%s@%s:%s' with connparm '%s'"
msgstr "No se pudo conectar a '%s@%s:%s' con connparm: '%s'"

#: providers/gda-primebase-server/gda-primebase-connection.c:163
#, c-format
msgid "%2ld: Connected to '%s@%s:%s' with connparm: '%s'"
msgstr "%2ld: Conectado a '%s@%s:%s' con connparm: '%s'"

#: providers/gda-primebase-server/gda-primebase-connection.c:171
msgid "Could not get PBIConnectionInfo(), disconnecting"
msgstr "No se pudo obtener PBIConnectionInfo(), desconectando"

#: providers/gda-primebase-server/gda-primebase-connection.c:176
#, c-format
msgid "%2ld: ConInfo: '%s@%s', Cl v%x, Srv '%s' v%x, Brand '%s'"
msgstr "%2ld: ConInfo: '%s@%s', Cl v%x, Srv '%s' v%x, Brand '%s'"

#: providers/gda-primebase-server/gda-primebase-connection.c:390
msgid "Not enough memory for primebase_Error structure."
msgstr "No hay suficiente memoria para primebase_Error structure."

#: providers/gda-primebase-server/gda-primebase-connection.c:397
#, c-format
msgid "%2ld: ERROR: %ld (%ld) : %s.\n"
msgstr "%2ld: ERROR: %ld (%ld) : %s.\n"

#: providers/gda-primebase-server/main-primebase.c:73
msgid "Could not register PRIMEBASE provider implementation"
msgstr "No se pudo registrarse implementación de proveedor PRIMEBASE"

#: providers/gda-tds-server/GNOME_GDA_Provider_TDS.oaf.in.h:1
msgid "GDA Datasource Access for Sybase/MS SQL"
msgstr "Acceso a la Fuente de Datos GDA para Sybase/MS SQL"

#: providers/gda-tds-server/GNOME_GDA_Provider_TDS.oaf.in.h:2
msgid "TDS provider connection factory"
msgstr ""

#: providers/gda-tds-server/gda-tds-connection.c:160
#, fuzzy
msgid "Not enough memory for tds_Connection"
msgstr "No hay suficiente memoria para primebase_Error structure."

#. Fixme: replace "dummy" with buf when CS_GET CS_SERVERNAME works
#: providers/gda-tds-server/gda-tds-connection.c:888
#, c-format
msgid "Reconnected to %s"
msgstr "Reconectado a %s"

#: providers/gda-tds-server/gda-tds-error.c:70
#, c-format
msgid "gda_tds_error_make(%p,%p,%p,%p)"
msgstr "gda_tds_error_make(%p,%p,%p,%p)"

#: providers/gda-tds-server/main-tds.c:78
msgid "Could not register gda-tds provider implementation"
msgstr "No se pudo registrar la implementación del proveedor gda-tds"

#: tools/gda-run.c:109
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: Entrada DSN desconfigurada"

#: tools/gda-run.c:124
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: fuente de datos '%s' no encontrada\n"

#: tools/templates/main-srv.c.tmpl:60
#, fuzzy
msgid "Could not register <PROVIDER-NAME> provider implementation"
msgstr "No se pudo registrarse implementación de proveedor PRIMEBASE"

#~ msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
#~ msgstr "impl_GDA_Connection_open: abriendo una conexión con DSN: %s"

#~ msgid "%s: an error ocurred while fetching data"
#~ msgstr "%s: ha ocurrido un error mientras se obtenían datos"

#~ msgid ""
#~ "OAF doesn't know about our IID; indicates broken installation. Exiting..."
#~ msgstr ""
#~ "OAF no conoce nuestro IID; indica que la instalación es incorrecta. "
#~ "Saliendo..."

#~ msgid ""
#~ "Another instance of this provider is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "Este proveedor ya ha sido registrado previamente en OAF. Saliendo..."

#~ msgid "Unknown error registering this provider with OAF. Exiting"
#~ msgstr "Error desconocido al registrar este proveedor en OAF. Saliendo"

#~ msgid "Registered with ID = %s"
#~ msgstr "Registrado con ID = %s"

#~ msgid "CORBA exception unloading report engine: %s"
#~ msgstr "Motor de informe de descarga de excepción CORBA: %s"

#~ msgid "GDA Distributed Report Engine"
#~ msgstr "Motor de Informe Distribuido GDA"

#~ msgid "Received signal %d, shutting down."
#~ msgstr "Recibida señal %d, cerrando"

#~ msgid "Received signal %d, shutting down cleanly"
#~ msgstr "Recibida señal %d, cerrando limpiamente"

#~ msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
#~ msgstr ""
#~ "Error a iniciar OAF: por favor, envía un mensaje de fallo a los "
#~ "mantenedores de OAF"

#~ msgid "Could not activate report engine"
#~ msgstr "No pudo activarse el motor de informe"

#~ msgid ""
#~ "Another instance of the report engine is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "Otra instancia de motor de informe ha sido ya registrada con OAF. "
#~ "Saliendo..."

#~ msgid "Unknown error registering report engine with OAF. Exiting"
#~ msgstr "Error desconocido registrando motor de informe con OAF. Saliendo"

#~ msgid "Registered report engine with ID = %s"
#~ msgstr "Motor de informe registrado con ID = %s"

#~ msgid "%s: length = %d"
#~ msgstr "%s: longitud = %d"

#~ msgid "%s: sending field description for '%s'"
#~ msgstr "%s: enviando la descripción del campo '%s'"

#~ msgid "schema_tables: table name = '%s'"
#~ msgstr "schema_tables: nombre de tabla = '%s'"

#~ msgid "schema_tables: table_schema = '%s'"
#~ msgstr "schema_tables: table_schema = '%s'"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
#~ msgstr "schema_procedures: proc_catalog = ¡'%s' NO USADO!\n"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
#~ msgstr "schema_procedures: proc_catalog = ¡'%s' NO USADO!"

#~ msgid "database name"
#~ msgstr "nombre de base de datos"

#~ msgid "DATABASE"
#~ msgstr "DATABASE"

#~ msgid "user name"
#~ msgstr "nombre de usuario"

#~ msgid "USER-NAME"
#~ msgstr "NOMBRE-DE-USUARIO"

#~ msgid "password"
#~ msgstr "contraseña"

#~ msgid "PASSWORD"
#~ msgstr "CONTRASEÑA"

#~ msgid "file name"
#~ msgstr "nombre de archivo"

#~ msgid "FILE-NAME"
#~ msgstr "NOMBRE-DE-ARCHIVO"
